Dockerfile

```diff
@@ -23,22 +25,27 @@ RUN apt-get update && apt-get install -y \
     \
     less \
     man \
-    vim
+    vim \
+    \
+    libssl-dev
+
+COPY dist/brainiak-* /mnt/brainiak
+
+WORKDIR /mnt/brainiak
 
-WORKDIR /mnt
+COPY tutorials/tutorials tutorials
 
 RUN set -e \
-    && python3 -m pip install --user -U pip \
-    && python3 -m pip install -U brainiak \
-    && python3 -m pip download --no-deps --no-binary :all: brainiak \
-    && export BRAINIAK_VERSION=$(basename brainiak-* .tar.gz | cut -b 10-) \
-    && tar -xf brainiak-*.tar.gz \
-    && for example in brainiak-$BRAINIAK_VERSION/examples/*/requirements.txt; \
+    && python3 -m pip install --user -U "pip<10" \
+    && python3 -m pip install --user -U . \
```

Comment:
For clarity I'd prefer to have WORKDIR /mnt and
&& python3 -m pip install --user -U brainiak \
&& python3 -m pip install --user -U -r brainiak/tutorials/requirements.txt \
&& for example in brainiak/examples/*/requirements.txt; \

Reply:
From the user perspective, I think WORKDIR should be `brainiak`, so users can easily find the tutorials and examples. As for clarity in the Dockerfile for developers, I think developers are used to working inside the `brainiak` directory.